packages/cli/lib/lib/webpack/preact-iso-prerender.js

```diff
@@ -0,0 +1,60 @@
+const { h, options, cloneElement } = require('preact');
```

Comment:
Vendoring `preact-iso`'s `prerender.js`: https://github.com/preactjs/wmr/blob/main/packages/preact-iso/prerender.js

`preact-iso` only offers ESM output and [dynamic imports cause v8's `vm` subsystem to segfault](https://github.com/nodejs/node/issues/35889). Probably need to have babel transpile `preact-iso` or something, at least for prerendering purposes.